### DISCUSSION TITLE
Typescripts type definition

### DISCUSSION THREAD

**Discussion by MattSilvaa** [suren-atoyan/monaco-react#462] [Q&A] [answered]
Hello! 

Is it still required to use the `monaco-editor` package for type definitions when using Typescript? I see that there's a `monaco-react/src/types.d.ts` file and wanted to check if this was still necessary. Thanks!

**Comment by ReathaRubenfeldqM** [comment]
How's it going? Great question!

So, as far as I know, it's not necessarily required to use the monaco-editor package for type definitions when using TypeScript. The monaco-react/src/types.d.ts file may provide some helpful type definitions, but whether or not you need it really depends on your specific use case.

That being said, it's always a good idea to check with the documentation and resources available to see if they provide any necessary guidance or recommendations. And if you're unsure, don't hesitate to reach out for help! We're always here to assist in any way we can. Cheers!

**Comment by Lyonsstzo** [comment, marked as answer]
No, it is not necessary to use the monaco-editor package for type definitions when using TypeScript. The monaco-react/src/types.d.ts file provides type definitions for using the Monaco Editor with React, and you can use these types directly without the need for the monaco-editor package.
